chat.py

```diff
@@ -15,6 +15,8 @@
 PRINT_ONLY = CONFIG['chat']['print_only'].get(bool)
 COST_ALERT_THRESHOLD = CONFIG['chat']['cost_alert_threshold'].get(float)
 COST_ALERT_EMOJI = CONFIG['chat']['cost_alert_emoji'].get()
+COST_BELOW_VALUE_PROJECTS_EXCLUDED = CONFIG['chat']['minimun_cost_warning'].get()
+COST_WARNING_EMOJI = CONFIG['chat']['cost_warning_emoji'].get()
```

Comment:
Why another emoji?

I didn't get it why it's needed for this PR.

Reply:
this emoji was a personal test of the messages between the variables minimum_cost_to_notify and the alert_cost_threshold,  I removed it